Client/src/components/App.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class App extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+
+        }
+    }
+    render() {
+      return (
+        <div>
+         < Overview />
+        </div>
+      );
+    }
+}
+
+
+class Overview extends React.Component {
```

Comment:
I would suggest putting this Overview component on a different file for separation of concerns.